charts/prometheus-windows-exporter/values.yaml

```diff
@@ -11,7 +11,7 @@ image:
   digest: ""
 
 config: |-
-  collectors:
+  collector:
```

Comment:
@jkroepke changed based on 0.30. release notes https://github.com/prometheus-community/windows_exporter/releases/tag/v0.30.0

Reply:
It's still collectors yet, see

https://github.com/prometheus-community/windows_exporter/blob/86e6d12518d2fa7f8f2f632634ade3aaa34fba22/config.yaml#L3-L5